src/components/RightPane/SectionTable/SectionTableBody.tsx

```diff
@@ -184,7 +207,15 @@ const LocationsCell = withStyles(styles)((props) => {
     );
 });
 
-const SectionEnrollmentCell = withStyles(styles)((props) => {
+interface SectionEnrollmentCellProps {
+    classes: ClassNameMap
+    numCurrentlyEnrolled: EnrollmentCount
+    maxCapacity: number
+    numOnWaitlist: number
+    numNewOnlyReserved: number
+}
+
+const SectionEnrollmentCell = withStyles(styles)((props: SectionEnrollmentCellProps) => {
     const { classes, numCurrentlyEnrolled, maxCapacity, numOnWaitlist, numNewOnlyReserved } = props;
 
     return (
```

Comment:
Adding this in reference to my previous comment, this is the closest line I am able to add a comment on. The code I am referring to is lines 229 and 230.

Reply:
Based on the original PR it was introduced in, I think the conditional is to prevent it displaying an empty string: https://github.com/icssc/AntAlmanac/pull/353#issuecomment-1104408864

Reply:
https://github.com/icssc/AntAlmanac/pull/423/commits/82fbf5440a7c458543c980093e00e0bb3d51942a